### PR TITLE
Feat: completed toast

### PR DESCRIPTION
**Pull request by vinc3nati**

### CODE REVIEW

File: components/Toast/toast.css [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+  color: var(--color-tertiary);
+  display: none;
+  z-index: 5;
+  border-radius: var(--border-radius-3);
+}
+
+.toast.outline {
+  background-color: var(--color-tertiary);
+  color: var(--color-default);
+  outline: var(--color-default) solid 1px;
```
Comment (PeeyushPrashant) [inline components/Toast/toast.css:17]: try using same unit throughout

**Review by PeeyushPrashant — APPROVED** [review]
good to go

File: components/Toast/toast.css [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+.btn-close {
+  background-color: transparent;
+  border: none;
+  outline: none;
+  cursor: pointer;
+  margin-left: 1rem;
+}
+
+.btn-close::before {
+  content: "\2715";
```
Comment (Rohanmond) [inline components/Toast/toast.css:74]: what is \2715 can u brief

**Review by Rohanmond — APPROVED** [review]

**Review by ramalingaa — APPROVED** [review]
No changes required

File: components/Toast/toast.css (hunk 2 above)
Comment (vinc3nati) [reply components/Toast/toast.css:74]: it's a html entity code for `x` i.e. close icon.

Comment (Rohanmond) [reply components/Toast/toast.css:74]: ohh got it

**Pull request merged**